go.mod

```diff
@@ -1,20 +1,16 @@
-module github.com/flatcar-linux/container-linux-config-transpiler
+module github.com/kinvolk/container-linux-config-transpiler
 
-go 1.15
+go 1.16
 
 require (
-	github.com/ajeddeloh/go-json v0.0.0-20160803184958-73d058cf8437 // indirect
+	github.com/ajeddeloh/go-json v0.0.0-20200220154158-5ae607161559 // indirect
```

Comment:
github.com/ajeddeloh/go-json redirects to https://github.com/coreos/go-json
Might be worth updating the dependency

Reply:
Thanks for suggesting @techplexEngineer. I think this project requires a lot more of maintenance tasks, which needs to be addressed, so perhaps we can address this as well at later stage.